### PR TITLE
[RFC] Show 2FA seed in case a user can't scan the QR code

### PR DESCRIPTION
**Pull request by bytehead**
See #77 

Use hex based secrets instead of binary. Users will be able to copy/paste this in case they can't use the QR code.

bildschirmfoto 2018-09-21 um 11 31 30

### CODE REVIEW

File: core-bundle/src/Resources/contao/languages/en/default.xlf [hunk 1]
```diff
@@ -1820,6 +1820,9 @@
       <trans-unit id="MSC.twoFactorEnforced">
         <source>Please enable up two-factor authentication before you continue.</source>
       </trans-unit>
+      <trans-unit id="MSC.twoFactorTextCode">
+        <source>If you can’t use a barcode, enter this text code instead.</source>
```
Comment (ausi) [inline core-bundle/src/Resources/contao/languages/en/default.xlf]: Shouldn’t this be *“If you can’t use a **QR code**, …”* instead?

Comment (bytehead) [reply core-bundle/src/Resources/contao/languages/en/default.xlf]: Yep... 😎

**Comment by leofeyer** [comment]
Nice 👍

**Comment by leofeyer** [comment]
Thank you @bytehead.

**Pull request merged**